### ISSUE TITLE
Connect to Chrome via port stored in DevToolsActivePort file

### ISSUE THREAD

**Issue by christian-bromann** [webdriverio/wdio-devtools-service#13]
## Problem

The latest Chromedriver version started to store the port of the devtools interface in the `DevToolsActivePort` instead of generating one randomly. We need to modify the connection process of the service to the browser.

**Comment by daviddekanter** [comment]
This seems to be a workaround:
- update chromedriver to verion 2.41
- add 'remote-debugging-port=9222' to chromeOptions

Example:
chromeOptions: {
      args: ['remote-debugging-port=9222'],
	}

**Comment by daviddekanter** [comment]
I think, since chromedriver v2.41, you can get the port by using this:  `browser.session().value['goog:chromeOptions'].debuggerAddress.split(":")[1]`

instead of:
```
browser.url('chrome://version')
browser.getText('#command_line').then((args) => parseInt(args.match(/--remote-debugging-port=(\d*)/)[1]))
```

**Comment by christian-bromann** [comment]
@daviddekanter do you mind to create a PR?

**Comment by daviddekanter** [comment]
@christian-bromann sure! no problem

**Comment by daviddekanter** [comment]
I think i need permission :)

**Comment by christian-bromann** [comment]
> I think i need permission :)

Not to create a PR 😉 . Just fork the repo and open a PR.

**Comment by christian-bromann** [comment]
Will be fixed by #11 and https://github.com/webdriverio/webdriverio/pull/2918
